src/wasm/kotlin/src/arcs/WasmJsInterop.kt

```diff
@@ -109,22 +117,22 @@ fun init(particlePtr: WasmAddress) {
 
 @Retain
 @ExportForCppRuntime("_syncHandle")
-fun syncHandle(particlePtr: WasmAddress, handlePtr: WasmAddress, encoded: WasmString) {
+fun syncHandle(particlePtr: WasmAddress, handlePtr: WasmAddress, encoded: WasmNullableString) {
     log("Getting handle")
     val handle = handlePtr.toObject<Handle>()
-    val encodedStr = encoded.toKString()
-    log("Handle is " + handle.name + "syncing '" + encodedStr + "'")
+    val encodedStr: String? = encoded.toNullableKString()
+    log("Handle is '${handle.name}' syncing '$encodedStr'")
     handle.sync(encodedStr)
     log("Invoking sync on handle on particle")
     particlePtr.toObject<Particle>().sync(handle)
 }
 
 @Retain
 @ExportForCppRuntime("_updateHandle")
-fun updateHandle(particlePtr: WasmAddress, handlePtr: WasmAddress, encoded1Ptr: WasmString,
-                 encoded2Ptr: WasmString) {
+fun updateHandle(particlePtr: WasmAddress, handlePtr: WasmAddress, encoded1Ptr: WasmNullableString,
```

Comment:
Style when arguments need to wrap is to place a newline after the opening `(`.

```kotlin
fun updateHandle(
    particlePtr: WasmAddress, handlePtr: WasmAddress,
    encoded1Ptr: WasmNullableString, encoded2Ptr: WasmNullableString
) {
    // body
}
```

Reply:
I tend to prefer putting all arguments on separate lines as well in this situation, but that's not a requirement of the style guide.